Composer/packages/types/src/settings.ts

```diff
@@ -67,6 +76,34 @@ export type DialogSetting = {
   botId?: string;
   skillHostEndpoint?: string;
   customFunctions: string[];
+  defaultLocale?: string;
+  runtimeSettings?: {
+    adapters?: AdapterRecord[];
+    features?: {
+      removeRecipientMentions?: boolean;
+      showTyping?: boolean;
+      traceTranscript?: boolean;
+      useInspection?: boolean;
+      blobTranscript?: {
+        connectionString?: string;
+        containerName?: string;
+      };
+      setSpeak: {
```

Comment:
`setSpeak` is not optional, but all of the properties in it are?

Reply:
Handled